src/js/app.js

```diff
@@ -0,0 +1,113 @@
+const $todoInput = document.querySelector("#new-todo-title");
+const $toggleInput = document.querySelector(`#todo-list`);
+const $todoCount = document.querySelector(`span strong`);
+
+$todoInput.addEventListener("keyup", addTodoItem);
+
+document.addEventListener("click", activeTodoItem);
+document.addEventListener("click", allTodoItem);
+document.addEventListener("click", finishTodoItem);
+
+function renderTodoItemTemplate(title) {
+    return ` <li class="active">
+                  <div class="view">
+                      <input class="toggle" onclick="onToggleTodoItem(event)" type="checkbox" >
+                      <label class="label" ondblclick="updateTodoItem(event)">${title}</label>
+                      <button class="destroy" onclick="deleteTodoItem(event)"></button>
+                  </div>
+                  <input class="edit" value=${title} />
+              </li>`;
+}
+
+function addTodoItem(event) {
+    const todoTitle = event.target.value;
+    const todoList = document.getElementById("todo-list");
+    if ("Enter" === event.key && "" !== todoTitle) {
+        todoList.insertAdjacentHTML("beforeend", renderTodoItemTemplate(todoTitle));
+        event.target.value = "";
+        count();
+    }
+}
+
+function onToggleTodoItem(event) {
+    const target = event.target;
+    if ("toggle" === target.className) {
+        if ("completed" === target.closest("li").className) {
+            target.closest("li").className = "active";
+        } else {
+            target.closest("li").className = "completed";
+        }
+    }
+}
+
+function updateTodoItem(event) {
+    const target = event.target;
+    if ("label" === target.className) {
+        target.closest("li").classList.add("editing");
+
+        const editInput = document.querySelector(".edit");
+        editInput.addEventListener("keydown", function(event){
+            if ("Escape" === event.key) {
+                target.closest("li").classList.remove("editing");
+            }
+
+            if ("Enter" === event.key) {
+                target.innerText = editInput.value;
+                target.closest("li").classList.remove("editing");
+            }
+        })
+    }
+}
+
+function deleteTodoItem(event) {
+    const target = event.target;
+    if ("destroy" === target.className) {
+        target.closest("li").remove();
+    }
+}
+
+function activeTodoItem(event) {
+    if (event.target.className === 'active') {
+        let countItems = 0;
+        const todoList = $toggleInput.children;
+        for (let i = 0; i < todoList.length; i++) {
+            if (todoList[i].classList.contains("completed")) {
+                todoList[i].style.display = "none";
+            } else {
+                countItems++;
+                todoList[i].style.display = "block";
+            }
+        }
+        $todoCount.innerHTML = countItems.toString();
+    }
+}
+
+function allTodoItem(event) {
+    if (event.target.className === 'all selected') {
```

Comment:
만약에 event.target의 class attribute 가  "selected all" 이면  조건문이  false가 되어서 조건문 내부가 실행되지 않을텐데, 이거는 의도하신 바가 아니시겠죠?

```suggestion
    if (event.target.classList.contains("all") && event.target.classList.contains("selected")) {
```

또한, JS에서는 string 리터럴을 만들때 '(Single Quote), "(Double Quote), `(backtick) 을 모두 사용가능한데요, 
하나로 일관성있게 작성하는 것이 좋습니다!

[심화] 사실 이러한 스타일링을 일관되게 해주는  prettier 라는 툴을 사용하시면, 신경안쓰셔도 되는 부분이긴 합니다 ㅎㅎ
https://prettier.io/